docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.7"
+
+services:
+
+  dev:
+    build:
+      context: .
+      target: dev-environment
+    depends_on:
+      - db
+    volumes:
+      - .:/work
+      - ruby-2.7.5-gem-cache:/usr/local/bundle
+      - ~/.docker-rc.d/:/.docker-rc.d/:ro
```

Comment:
Is `~/.docker-rc.d/` a standard directory that gets created automatically by all Docker installations? If it's not, we should probably leave this out (assuming that `docker-compose up` will fail if the directory doesn't exist).

Reply:
It's not a Docker standard, but a GreenSync standard. I can't remember how we came up with that exact name of `~/.docker-rc.d/.docker-bashrc`. When a path you're mounting into a conainer doesn't exist, Docker creates it as a directory. With that in mind, I thought it was relatively harmless to leave in. What do you think?

Reply:
That does sound harmless but we should at least document its usage if we're going to add it to Que.
Admittedly, most Que developers outside GS probably won't want to go to the trouble of setting up their aliases to work inside one Dockerised project, but it seems remiss not to mention it.